src/js/components/AppVersionComponent.jsx

```diff
@@ -224,9 +223,12 @@ var AppVersionComponent = React.createClass({
       ? <UnspecifiedNodeComponent />
       : getHighlightNode(appVersion.networks);
 
-    var urisNode = (appVersion.uris == null || appVersion.uris.length === 0)
+    const uris = appVersion.fetch.map((fetch) => {
+      return fetch["uri"];
+    });
+    var urisNode = (uris == null || uris.length === 0)
```

Comment:
We can remove the `uris == null` check as it will always be defined if fetch is defined.